### DISCUSSION TITLE
Juste a correction for the readme

### DISCUSSION THREAD

**Discussion by rrkheloufi** [gmonarque/go-json-translate#6] [Ideas]
Hello ! 
First of all, thank you so much for your program, It works perfectly. One command and bam : a new trad. 
So, thank you !!

If i can just give you something, in the README, you should update cd `go-json-translate` to `cd go-json-translate/src`
As a non go user, I had to search why the command didn't work from the go-json-translate folder. 

But that's it ! Otherwise, amazing 😄

**Comment by kasterby** [comment]
Hello,

Thank you for your feedback, I'm glad you found this little tool useful!
I've updated the readme as you recommended.

Cheers,
Guillaume
